### DISCUSSION TITLE
It doesn't show the music I'm listening to on discord

### DISCUSSION THREAD

**Discussion by Pableske** [pear-devs/pear-desktop#1588] [Q&A] [answered]
It doesn't show the music I'm listening to on discord,I already linked the account. Help please

**Comment by JellyBrick** [comment, marked as answer]
https://github.com/th-ch/youtube-music/issues/1524#issuecomment-1860515627

![image](https://github.com/th-ch/youtube-music/assets/16558115/14c0a6b2-cea6-4ad0-b776-96a183a47ea5)

It works well.

**Reply by Pableske** [reply]
I do not know what I'm doing wrong :(

**Reply by Pableske** [reply]
Fixed up

**Reply by fourmicronde** [reply]
how did u fixed it ?

**Reply by VanGeko** [reply]
for me its also not working.

**Reply by RTels** [reply]
3.7.2.0

Not working on my end as well.
![imagem_2025-01-24_130458323](https://github.com/user-attachments/assets/52b7bdeb-44c5-4398-a99d-742a1f125e90)
![imagem_2025-01-24_130517367](https://github.com/user-attachments/assets/c37cb587-7ff6-4bb9-80bf-29830054c18a)

**Reply by itsGxbe** [reply]
was working for me well last night before I turned off my PC, now its not even popping up

**Reply by RTels** [reply]
just discovered this app and everythings works great, except that. hope it gets fixed!

**Reply by itsGxbe** [reply]
been using for monthss now, i love it bro, no ads and all the tweaks, def worth it, and yea hopefully it gets fixed soon, it usually works perfectly

**Reply by RTels** [reply]
@itsGxbe update. Tried Spotify and the same thing happened. Some bonkers on discords end. You should be good now. Working great!

**Reply by itsGxbe** [reply]
ohh alright then yea something was wrong with discords rich presence. and bet bro! that’s good, thank you for letting me know!

**Reply by JellyBrick** [reply]
@RTels @itsGxbe See #2914

**Reply by itsGxbe** [reply]
@JellyBrick its working now, thanks bro!
